content/en/agent/guide/podman-support-with-docker-integration.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using the Docker integration with Podman container runtime
+kind: documentation
+---
+
+Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Read more on [https://podman.io/][1].
+
+Podman is an alternative to Docker as it provides a Docker-compatible CLI interface and socket. This specificity allows to use the Datadog Agent Docker integration with Podman containers.
+
+## Requirements
+
+* Podman version >= 3.2.0
+* Podman configured to expose its communication socket.
+* Datadog Agent version >= 7.30.0
+
+## Agent deployment as a podman container
+
+To deploy the agent as a podman container, the command to run is very similar to the one used for [docker][2].
+
+```
+$ podman run -d --name dd-agent \
+    -v /run/podman/podman.sock:/run/podman/podman.sock:ro \
+    -v /proc/:/host/proc/:ro \
+    -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+    -e DD_API_KEY=<API_KEY> \
+    -e DOCKER_HOST=unix:///run/podman/podman.sock \
+    --privileged \
```

Comment:
I think the need of `--privileged` depends on the OS, maybe we should a note about it?

Reply:
maybe more about the feature enabled like (system-probe)